src/client/components/common/Header/index.tsx

```diff
@@ -0,0 +1,155 @@
+
+import styles from './Header.module.scss';
+import Link from 'next/link';
+import Button from '../Button';
+import { useReset } from '../../../hooks/useReset/useReset';
+import { Tooltip } from '@mui/material';
+
+interface HeaderProps {
+  notificationBar?: {
+    arrowText?: string;
+    barBody?: string;
+    url?: string;
+    backgroundColor?: string;
+  };
+  darkMode?: boolean;
+}
+export default function Header({ notificationBar, darkMode }: HeaderProps) {
+  const [isMobileMenuOpen, setIsMobileMenuOpen] = useState(false);
+
+  useEffect(() => {
+    const mobileBodyClass = 'isMobileMenuOpen';
+    if (isMobileMenuOpen) {
+      document.body.classList.add(mobileBodyClass);
+    } else {
+      document.body.classList.remove(mobileBodyClass);
+    }
+  }, [isMobileMenuOpen]);
+
+  useEffect(() => {
+    Prism.highlightAll();
+  }, []);
+
+  const handleToggleMobileMenu = () => {
+    setIsMobileMenuOpen(!isMobileMenuOpen);
+  };
+  const { mutate } = useReset({});
+
+  return (
+    <>
+      {notificationBar && !isMobileMenuOpen && (
+        <HeaderBar
+          linkUrl={notificationBar.url}
+          arrowText={notificationBar.arrowText}
+          backgroundColor={notificationBar.backgroundColor}
+        >
+          {<div dangerouslySetInnerHTML={{ __html: notificationBar.barBody ?? '' }} />}
+        </HeaderBar>
+      )}
+      <header className={classNames(styles.header, { [styles.darkHeader]: darkMode })}>
+        <div className={styles.nav}>
+          <Container size="large" className={styles.root}>
+            <nav className={styles.navMain}>
+              <div className={styles.navLeft}>
+                <Link href="/" className={styles.link} title="Logo">
+                  {darkMode ? (
+                    <Image src={LogoDarkSvg} className={styles.logo} alt="Fingerprint logo" />
+                  ) : (
+                    <Image src={LogoSvg} className={styles.logo} alt="Fingerprint logo" />
+                  )}
+                </Link>
+                <DropdownMenu
+                  darkMode={darkMode}
+                  name="Use cases"
+                  className={styles.desktopOnly}
+                  dropdownProps={{
+                    darkMode,
+                    leftColumns: [
+                      {
+                        list: USE_CASES.slice(0, 3),
+                        cardBackground: true,
+                      },
+                      {
+                        list: USE_CASES.slice(3),
+                        cardBackground: true,
+                      },
+                    ],
+                  }}
+                />
+
+                <DropdownMenu
+                  darkMode={darkMode}
+                  name="Platform"
+                  className={styles.desktopOnly}
+                  dropdownProps={{
+                    darkMode,
+                    leftColumns: [{ list: [{ title: 'Playground', url: '/playground' }], cardBackground: true }],
```

Comment:
I saw it in the `MobileNavbar` before (`{ title: 'Playground', url: '/playground' }`)

Reply:
True, moved to `content.ts` and imported from there in both places. https://github.com/fingerprintjs/fingerprintjs-pro-use-cases/pull/66/commits/3dad5877f4994ead6421bef5a6e5d20a78a6eb87